Sources/IO/Geometry/GLTFImporter/index.js

```diff
@@ -1,25 +1,147 @@
 import macro from '@kitware/vtk.js/macros';
+import DataAccessHelper from 'vtk.js/Sources/IO/Core/DataAccessHelper';
+import Constants from 'vtk.js/Sources/IO/Geometry/GLTFImporter/Constants';
+
+// Enable data soure for DataAccessHelper
+import 'vtk.js/Sources/IO/Core/DataAccessHelper/LiteHttpDataAccessHelper'; // Just need HTTP
+
+// ----------------------------------------------------------------------------
+// vtkGLTFImporter methods
+// ----------------------------------------------------------------------------
 
 function vtkGLTFImporter(publicAPI, model) {
   //
   model.classHierarchy.push('vtkGLTFImporter');
+
+  if (!model.dataAccessHelper) {
+    model.dataAccessHelper = DataAccessHelper.get('http');
+  }
+
+  // To support destructuring
+  if (!model.compression) {
+    model.compression = null;
+  }
+
+  if (!model.progressCallback) {
+    model.progressCallback = null;
+  }
+
+  function fetchData(url, option = {}) {
+    const compression =
+      option.compression !== undefined ? option.compression : model.compression;
```

Comment:
you can use optional coalescing operator `option.compression ?? model.compression`

Reply:
should be fixed